src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -132,19 +128,6 @@ const AdminDashboard = () => {
     }
   };
 
-  const changeSelected = incident => {
-    if (!confirmApprove && !confirmReject) {
-      if (selected.includes(incident.incident_id)) {
-        const newSelected = selected.filter(id => {
-          return id !== incident.incident_id;
-        });
```

Comment:
I know this is unused, but isn't it really just un-implemented? Seems like it allows you to change the selected record to then perform actions upon it?

Reply:
Similar to what Ahmed commented. I also think it would be good to understand, if we can, the original intent of the developer. Perhaps it was just redundancy.

Reply:
I would also love to see/hear more of an explanation regarding why we _wouldn't_ use this function - I know you've gotten to know this portion of the codebase, and it would help validate the decision to remove it to hear your thoughts!